_posts/pl/2021-12-17-frontend-developer.md

```diff
@@ -0,0 +1,157 @@
+- Budowanie kreatora wieloetapowej akceptacji operacji asynchronicznych:
+    - przygotowanie generycznego mechanizmu wieloetapowej akceptacji na wiele rąk,
+    - referencyjna implementacja funkcjonalności potwierdzania paczek transakcji,
+    - uogólnienie kodu na potrzeby przyszłych funkcjonalności i przygotowanie funkcjonalnych prototypów Storybook.
+- Komponent tworzenia wiadomości oferujący funkcjonalność formatowania treści:
+    - opracowanie komponentu edytora WYSIWYG z przykładami w UI guideline,
+    - wykorzystanie komponentu na ekranie tworzenia wiadomości,
+    - integracja wysyłki z istniejącą usługą REST z uwzględnieniem praktyk UI/UX i optymistycznego potwierdzania operacji.
+
+I tak dalej. To tylko kilka z przykładów kompetencji i umiejętności których szukamy u osób specjalizujących się w obszarze Frontend Developer.
+
+## Czym można się zająć w Consdata?
+
+W Consdata, w zależności od zespołu, możesz pracować przy jednej lub kilku z naszych kluczowych aplikacji web. W kolejnych punktach przejrzymy razem co, i w jakiej skali, budujemy dla naszych klientów.
+
+W zależności od wybranego projektu, pojawiające się w nim wyzwania będą bliżej domeny lub stosowanej technologii.
+
+![Projekty Consdata](/assets/img/posts/2021-12-17-frontend-developer/os-projektow.webp)
+<span class="img-legend">Projekty Consdata z uwzględnieniem charakterystyki nacisku na domenę lub technologię</span>
+
+### iBiznes
+
+iBiznes to złożony system bankowości elektronicznej do obsługi klienta korporacyjnego. Głównym wyznacznikiem projektu jest jego złożoność biznesowa.
+
+Mimo że dziedzina stanowi główne wyzwanie, to jednak nie brakuje w nim wyzwań technologicznych, takich jak np. wielomodułowa architektura zbudowana z myślą o niezależnych wdrożeniach obszarów funkcjonalnych, równoczesna praca wielu klientów nad pojedynczymi obiektami domeny czy optymalizacja z badaniami UX/UI i dbanie o czytelność złożonych biznesowo operacji.
+
+Jeżeli szukasz wyzwania w projekcie, który skupia się na domenie i jego złożoność leży w implementowanych funkcjonalnościach, to iBiznes jest właśnie dla Ciebie!
+
+iBiznes w liczbach to ~300k linii kodu TypeScript i ~30k linii SASS samego frontendu napisanego w Angularze!
+
+### Mailbox
+
+Skłaniasz się bardziej w stronę *cutting edge* technologii, o których tyle słyszysz na konferencjach? Twój wzrok powinien paść na projekty Mailbox. Nie da się mówić o Mailbox nie mówiąc o skali! Codzienna komunikacja z milionami klientów uczy nowego spojrzenia na nawet najprostsze problemy (zdziwiłbyś się ile daje optymalizacja aktualizacji licznika nieprzeczytanych wiadomości o, w praktyce, nieograniczonej liczbie użytkowników).
+
+Udział w projekcie Mailbox oferuje zrównoważoną dawkę złożoności biznesowej i wyzwań technologicznych.
+
+Dodatkowym atutem jest możliwość pochwalenia się znajomym i rodzinie co się ostatnio robiło, wystarczy, że są klientami jednego z naszych klientów, a że klienci duzi to i szansa na bycie ich klientem spora ;-)
```

Comment:
_Dodatkowym atutem jest możliwość pochwalenia się znajomym i rodzinie co się ostatnio robiło, wystarczy, że są klientami jednego z banków z którymi współpracujemy. Są to najbardziej popularne banki, także szansa na bycie ich klientem jest spora ;-)_